charts/cass-operator/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ClusterRoleBinding
```

Comment:
Should this have the same: {{- if .Values.clusterScoped }} ? Or does it not matter if there's no ClusterRole for it?

Reply:
Is this extra file? There's also "rolebinding.yaml" that includes the same ClusterRole?

Reply:
This ClusterRole is needed all the time. I know it is needed for decommission support to  work with persistent volumes which are cluster scoped. See https://github.com/datastax/cass-operator/pull/242. 

I it also mentions nodes. I will have to see what kind of calls are made for nodes, and then try to rename this to something a bit more intuitive.

Reply:
I added a comment in `clusterrole.yaml` explaining why it is needed.